### PR TITLE
aseprite.skia: cleanup, switch to pname + version, add meta

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131250` at 4449d867 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/4711604e81ac91e885b347b2b73ce163">aseprite-unfree</a></li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

File: pkgs/applications/editors/aseprite/skia.nix [hunk 1]
```diff
@@ -73,4 +74,12 @@ stdenv.mkDerivation {
       third_party/externals/angle2/include \
       third_party/skcms/**/*.h
   '';
+
+  meta = with lib; {
+    description = "Skia is a complete 2D graphic library for drawing Text, Geometries, and Images";
+    homepage = "https://skia.org/";
+    license = licenses.bsd3;
+    maintainers = with maintainers; [ ];
+    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/aseprite/skia.nix:83]:
```suggestion
    platforms = platforms.all;
    # make fails with missing rule how to make a target
    broken = true;
```

File: pkgs/applications/editors/aseprite/skia.nix [hunk 2]
```diff
@@ -33,7 +34,7 @@ stdenv.mkDerivation {
     mesa libX11
   ];
 
-  preConfigure = with depSrcs; ''
+  postUnpack = with depSrcs; ''
```
Comment (Artturin) [inline pkgs/applications/editors/aseprite/skia.nix:37]: build fails because of this

Comment (Artturin) [reply pkgs/applications/editors/aseprite/skia.nix:37]:
```suggestion
  postPatch = with depSrcs; ''
``` 
this works

Comment (Artturin) [reply pkgs/applications/editors/aseprite/skia.nix:37]:
```
  postUnpack = with depSrcs;
    mkdir -p source/third_party/externals
    ln -s ${angle2} source/third_party/externals/angle2
    ln -s ${dng_sdk} source/third_party/externals/dng_sdk
    ln -s ${piex} source/third_party/externals/piex
    ln -s ${sfntly} source/third_party/externals/sfntly
  '';

**Pull request merged**